src/fragments/lib-v1/graphqlapi/js/upgrade-guide.mdx

```diff
@@ -55,7 +55,9 @@ API.graphql({
 });
 ```
 
+<InlineFilter filters={['javascript', 'angular', 'nextjs', 'react', 'vue']}>
 ### Optimistic UI and cached data revalidation
+</InlineFilter>
```

Comment:
Sorry if I'm missing something, but is this inline filter supposed to wrap around the heading only? Should it be wrapping around the text too?